sqldelight-gradle-plugin/src/main/kotlin/app/cash/sqldelight/gradle/SqlDelightDatabase.kt

```diff
@@ -12,19 +12,19 @@ import org.gradle.api.GradleException
 import org.gradle.api.Project
 import org.gradle.api.file.DirectoryProperty
 import org.gradle.api.internal.catalog.DelegatingProjectDependency
-import org.gradle.api.provider.ListProperty
 import org.gradle.api.provider.Property
 import org.gradle.api.provider.Provider
 import java.io.File
 import javax.inject.Inject
+import org.gradle.api.file.ConfigurableFileCollection
 
 abstract class SqlDelightDatabase @Inject constructor(
   val project: Project,
   val name: String,
 ) {
 
   init {
-    sourceFolders.convention(listOf("sqldelight"))
+    srcDirs.setFrom(emptyList<String>())
```

Comment:
Nit, you don't need this, the srcDirs is empty by default

Reply:
Changed and TIL